src/app/models/body-type.ts

```diff
@@ -2,3 +2,17 @@ export interface BodyType {
   code: string;
   description: string;
 }
+
+export interface IBody {
```

Comment:
lets remove this interface. They are all fields in TechRecord

Reply:
used as temporary interface to just pass down what we need instead of the entire technicalRecord so that it will break if this changes. I wanted to avoid passing down unnecessary things as this can be error prone especially when some props have very similar names.
The shape of the reactive form will remain the same
```
techRecord: {
  make,
  function,
  model,
   ....
  bodyType: {
    code,
    description
  }
}
```